spec/controllers/polls_controller_spec.rb

```diff
@@ -139,11 +184,33 @@
   describe 'GET #edit' do
     let(:poll) { FactoryGirl.create(:valid_poll) }
 
-    it 'should be success' do
+    it 'should be succes' do
       expect(get :edit, id: poll.id).to be_succes
     end
     it 'should render template edit' do
       expect(get :edit, id: poll.id).to render_template(:edit)
     end
   end
+
+  describe 'POST #make_choise' do
+    before(:each) { @poll = FactoryGirl.create(:valid_poll) }
```

Comment:
Please never ever use `@variables` in your specs.
It's a really bad practice, and will burn you badly as soon as you will try to run your specs in several parallel threads.

Use `let` instead.